docs/user/notifications.rst

```diff
@@ -0,0 +1,16 @@
+Notifications
```

Comment:
As I mentioned in another comment, I think we can move this content to https://docs.readthedocs.com/en/stable/versions.html#version-warning instead of creating a new page.

The "Pull request notification" is already mentioned in `/pull-requests` page, so we can move only the other two that are related to _versions_ in particular.

Reply:
I could go both ways. I think it's useful to note the features here at a high level, but also fine keeping them inside the feature they're adding notifications to. I don't see us really pointing people to this page as is though, so probably easiest to keep it in the feature-specific section for now?